serverless/knative_eventing/serverless-kn-trigger.adoc

```diff
@@ -18,8 +18,25 @@ Before you can use triggers, you will need:
 
 * Knative Eventing and `kn` installed.
 * An available broker, either the `default` broker or one that you have created.
++
+You can create the `default` broker either by following the instructions on xref:../serverless-using-brokers.adoc#serverless-using-brokers[Using brokers with Knative Eventing], or by using the `--inject-broker` flag while creating a trigger. Use of this flag is described in the procedure below.
 * An available event consumer, for example, a Knative service.
 
+[id="create-kn-trigger"]
+== Creating a trigger using `kn`
+
+.Procedure
+
+To create a trigger, enter the following command:
+----
+$ kn trigger create <TRIGGER-NAME> --broker <BROKER-NAME> --filter <KEY=VALUE> --sink <SINK>
+----
+
+To create a trigger and also create the `default` broker using broker injection, enter the following command:
+----
+$ kn trigger create <TRIGGER-NAME> --broker --inject-broker --filter <KEY=VALUE> --sink <SINK>
```

Comment:
This would fail with " Invalid value: "--inject-broker"" ,  because the --broker expects the name of the broker, If we want to use the default, we need to omit the "--broker" flag altogether.

kn trigger create <TRIGGER-NAME> --inject-broker --filter <KEY=VALUE> --sink <SINK>

Reply:
or, alternatively, include the "--broker default" as well:

kn trigger create --broker default --inject-broker --filter <KEY=VALUE> --sink